dataland-frontend/src/services/ApiClients.ts

```diff
@@ -63,8 +132,9 @@ export class ApiClientProvider {
     return getUnifiedFrameworkDataControllerFromConfiguration(framework, configuration);
   }
 
-  async getMetaDataControllerApi(): Promise<MetaDataControllerApiInterface> {
-    return this.getConstructedApi(MetaDataControllerApi);
+  async getMetaDataControllerApi(): Promise<backendApis.MetaDataControllerApiInterface> {
+    //TODO this is a backend controller, why needed?
```

Comment:
@MarcTM01 Could you please have a look at this TODO review question from Emanuel. I have no knowledge on the matter.

Reply:
Sure. I think the TODO is referring to the fact that a metaDataController is already included in the `readonly backendClients: ApiBackendClients` object. Therefore, this function is redundant.

Reply:
Done.

Reply:
Were all those methods unnecessary?
![image](https://github.com/d-fine/Dataland/assets/104447806/ac9df5b9-1210-4cc9-ba0e-8a6b831fbc46)

Reply:
ok